tests/unit/test_sqlite_graph.py

```
@@ -373,3 +373,70 @@ async def slow_inner(*args, **kwargs):
+        await adapter.create_edge("c", "d", "REF", {})
+
+        # a, b, c の間のエッジのみ取得 (a->b, b->c)
+        edges = await adapter.list_edges_for_memories(["a", "b", "c"])
+        assert len(edges) == 2
+        edge_pairs = {(e.from_id, e.to_id) for e in edges}
+        assert ("a", "b") in edge_pairs
+        assert ("b", "c") in edge_pairs
+        assert ("c", "d") not in edge_pairs
+
+    async def test_list_edges_for_memories_chunking(self, adapter: SQLiteGraphAdapter) -> None:
+        """大量のメモリID（900件超え）を指定しても正常に動作する (chunking)."""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)